src/coreclr/gc/gc.cpp

```diff
@@ -13587,6 +13617,40 @@ void gc_heap::distribute_free_regions()
 #endif //USE_REGIONS
 }
 
+void gc_heap::age_free_regions(const char* label)
+{
+#ifdef MULTIPLE_HEAPS
+    for (int i = 0; i < gc_heap::n_heaps; i++)
+    {
+        gc_heap* hp = gc_heap::g_heaps[i];
+#else //MULTIPLE_HEAPS
+    {
+        gc_heap* hp = pGenGCHeap;
+        const int i = 0;
+#endif //MULTIPLE_HEAPS
+
+        hp->descr_generations (label);
+#ifdef USE_REGIONS
```

Comment:
we'd want to leave this in gc1 because this doesn't only apply to regions

Reply:
Interestingly, I had preserved the non-regions behavior by having the `#ifdef USE_REGIONS` scoped carefully here, but it was confusing because of the name `age_free_regions` (which is already a good reason to pull this line out).  But then, after pulling this line out, since the entire method is now only for regions, I don't need an extra `ifdef` to hoist out the code for your next comment.